engine/core/IgeEntity.js

```diff
@@ -89,7 +89,7 @@ var IgeEntity = IgeObject.extend({
 	},
 
 	isRendering() {
-		return ige.isClient && !!ige.entitiesToRender.trackEntityById[this.id()];
+		if (ige.isClient) return !!ige.entitiesToRender.trackEntityById[this.id()];
```

Comment:
and this will return boolean for client and `undefined` for server, so we have inconsistent return type

Reply:
yeah I was thinking about return type. If we want to take that into consideration, then we should probably just add `else return false`